Sources/Benchmark/BenchmarkReporter.swift

```diff
@@ -14,6 +14,17 @@
 
 import Foundation
 
+extension String {
+    func leftPadding(toLength: Int, withPad: String) -> String {
+        let stringLength = self.count
+        if stringLength < toLength {
+            return String(repeating:withPad, count: toLength - stringLength) + self
+        } else {
+            return String(self.suffix(toLength))
```

Comment:
Hmmm, this seems a little dangerous... since we're implementing our own padding logic, could we perhaps trigger a fatal error here (given that we know we should have already computed the max of the column anyway)?

Reply:
@saeta  Should we trigger  `fatalError()` in a block of else, right?

Reply:
Yes, I believe so (as that's preferable to a truncation which could yield misleading printouts), and (in this case) indicates a logic bug in another function in this file.

Reply:
Done.